src/scss/common/responsive.scss

```diff
@@ -0,0 +1,43 @@
+@media screen and (min-width: 576px){
+  .container {
+    max-width: 540px;
+  }
+}
+@media screen and (min-width: 768px){
```

Comment:
For a mix of min and max width it's recommended to follow breakpoints guidelines:
@media (min-width: 769) {}
@media (max-width: 768) {}

In case there is the same value for min and max width when window reaches that size both rules will be applied.